src/les2/Main.java

```diff
@@ -0,0 +1,51 @@
+package les2;
+
+import java.util.Random;
+
+public class Main {
+
+    int maxSize = 100000;
+    //int maxSize = 1000000;
+    Random random = new Random();
+    MyArraylist<Integer> mal = new MyArraylist<>();
+    MyArraylist<Integer> mal1 = new MyArraylist<>();
+    MyArraylist<Integer> mal2 = new MyArraylist<>();
+
+    public static void main(String[] args) {
+        Main d = new Main();
+        d.mainStart();
+    }
+
+    public void mainStart(){
+        for (int i = 0; i < maxSize; i++) {
+            mal.add(random.nextInt(maxSize));
+            mal1.add(random.nextInt(maxSize));
+            mal2.add(random.nextInt(maxSize));
+        }
```

Comment:
```suggestion
        for (int i = 0; i < maxSize; i++) {
            int value = random.nextInt(maxSize);
            mal.add(value);
            mal1.add(value);
            mal2.add(value );
        }
```
чтобы сортировки проводились на одном и том же наборе данных